client/components/social-buttons/google.js

```diff
@@ -56,6 +62,14 @@ class GoogleLoginButton extends Component {
 			).catch( error => {
 				this.initialized = null;
 
+				if ( 'idpiframe_initialization_failed' === error.error ) {
+					// This error is caused by 3rd party cookies being blocked.
+					this.setState( { error: translate( 'Your privacy settings are blocking us from connecting to your Google account. ' +
+						'Please enable "third-party cookies" in your browser or operating system, or log in with an email address instead.'
+					) } );
+					return null;
```

Comment:
I think it's fine to show an error in the console here. if we want to hide it in a `debug()` call for instance, that should be done by calling `catch` on the promise returned by `this.initialize()`.